src/main/webapp/sign-up.html

```diff
@@ -40,12 +40,12 @@
           <h2>get notified with emails</h2>
           <form action="/user" method="POST" autocomplete="off">
             <label class="input-name">name</label>
-            <input type="text" name="name" required>
+            <input type="text" name="name" maxlength = "30" required>
```

Comment:
Haha nice this way we won't get any movie scripts

Reply:
Yes! LOL